tidy3d/components/fitter/fit.py

```diff
@@ -614,7 +612,7 @@ def from_url(
         :class:`DispersionFitter`
             A :class:`DispersionFitter` instance.
         """
-        resp = requests.get(url_file, verify=Env.current.ssl_verify)
+        resp = requests.get(url_file)
```

Comment:
I'm not sure about this change. It works fine without verify. From git history, it was added by MC. Can we still call `Env` in the main module?